libconfig/src/config_parser.c

```diff
@@ -469,171 +559,101 @@ static void config_validate(config_parser_type * config, config_content_type * c
 */
 
 
-static void config_parse__(config_parser_type * config ,
-                           config_content_type * content ,
-                           path_stack_type * path_stack ,
-                           const char * config_input ,
-                           const char * comment_string ,
-                           const char * include_kw ,
-                           const char * define_kw ,
+static void config_parse__(config_parser_type * config,
```

Comment:
Okay, this `config_parse__` is quite a handful.  Could we possible get a short description of its purpose and behavior above it?

Reply:
Okay, nevermind. I found it.  Was was located above it.